app/javascript/controllers/reveal_controller.js

```diff
@@ -0,0 +1,10 @@
+// reveal_controller.js
```

Comment:
Sorry for the confusion @linkonsat, I know you got these comments from the example code, I should have made it clear the comments were annotations. The code should be clear enough without them though. Can we remove the comments in the js and HTML file please?